pkgs/tools/virtualization/nixos-container/nixos-container.pl

```diff
@@ -22,6 +22,7 @@ sub showHelp {
        nixos-container destroy <container-name>
        nixos-container start <container-name>
        nixos-container stop <container-name>
+       nixos-container kill <container-name> -- args...
```

Comment:
I would omit `args` because that exposes the `machinectl` interface, which is an implementation detail.